ModelBundle/Repository/ContentRepository.php

```diff
@@ -66,37 +66,59 @@ public function findLastPublishedVersionByContentIdAndLanguage($contentId, $lang
     }
 
     /**
+     * @deprecated use findByContentTypeAndChoiceTypeAndKeywordsAndLanguage
+     *
+     * @param string $language
      * @param string $contentType
      * @param string $choiceType
      * @param string $keywords
      *
      * @return array
      */
-    public function findByContentTypeAndChoiceTypeAndKeywords($contentType = '', $choiceType = self::CHOICE_AND, $keywords = null)
+    public function findByContentTypeAndChoiceTypeAndKeywords($language = null, $contentType = '', $choiceType = self::CHOICE_AND, $keywords = null)
```

Comment:
signature error (comparing with https://github.com/open-orchestra/open-orchestra-model-interface/pull/64/files?diff=unified). Are you sure it's relevant add $language in this case ?